akka-edge-docs/src/main/paradox/feature-summary.md

```diff
@@ -16,7 +16,7 @@ delivery of messages.
 
 Projections over gRPC uses the event journal on the producer side and Akka Projections event processing and offset
 tracking on the consumer side. The transparent data transfer between producers and consumers is implemented with
-Akka gRPC.
+gRPC.
```

Comment:
Might be fine, but just to note, we do think of it as an _internal_ gRPC protocol though, not really something we've documented to the point where there easily be third party implementations that we are not taking part in.

Reply:
We're now using [Tonic](https://github.com/hyperium/tonic) internally within akka-edge-rs. Hence this change.